ocs_ci/utility/templating.py

```diff
@@ -136,7 +136,7 @@ def dump_to_temp_yaml(src_file, dst_file, **kwargs):
         yaml.dump(data, yaml_file)
 
 
-def load_yaml_to_dict(file, multi_document=False):
+def load_yaml(file, multi_document=False):
```

Comment:
the name sounds like synonym for 'yaml.load' , I think we should use better internal name that represents it returning dict or dict generator, eg: `load_yaml_to_dict_gen`, Frankly speaking the current name doesnt have to change , there isn't that much confusion.

Reply:
If we are not using jinja2 templating now, maybe we could revisit this with yaml.load() as is.